egs2/mucs21_subtask1/asr1/conf/train2.yaml

```diff
@@ -0,0 +1,60 @@
+# network architecture
```

Comment:
I appreciate your effort of having different architectures, but could you arrange them in `conf/tuning/train_{conformer, transformer}.yaml` and then use a symlink as `train.yaml` for the best architecture? You could refer to this (https://github.com/espnet/espnet/tree/master/egs2/commonvoice/asr1/conf)